### PR TITLE
Adopt browser-level image lazy-loading

### PR DESCRIPTION
**Pull request by taylrj**
Address [TWREPORTER-426](https://twreporter-org.atlassian.net/browse/TWREPORTER-426)

### CODE REVIEW

**Review by duidae — COMMENTED** [review]
mostly looking good & 2 little questions:

1. curious why editor-picks/fullscreen/normal/pink don't need to apply lazyloading?
2. what's the pagespeed outcome when applied this nice improvement?

thanks!

**Comment by taylrj** [comment]
> why editor-picks/fullscreen/normal/pink don't need to apply lazyloading?

The reasons why I think there's no need to apply lazyload to the leading image of article page in fullscreen/normal/pink theme and the image in editor-picks are:

1. In most case,  it is the largest image and should display ASAP to not lower the LCP (largest contentful paint) score
2. usually, users visit the web pages and see the images at first sight, hence it do not need to be lazyloaded

Do you agree with above ? Please share your opinion if you have different point of view, thanks @duidae.

**Review by duidae — APPROVED** [review]
no problem, LGTM!

**Comment by taylrj** [comment]
> 2. what's the pagespeed outcome when applied this nice improvement?

#TL;DR
- the score is almost the same after this change 🥲
- lets look into the opportunity `Defer offscreen images` in the reports 
   - index page: 
     - desktop: save 1.8s (reduce estimated savings from 1.96 to 0.16s)
     - mobile: save 19.96s (reduce estimated savings from 20.26s to 0.3s)
   - article page (slug: russian-invasion-of-ukraine-2022-kid-frequently-asked-questions):
     - desktop: save 2.8s (reduce estimated savings from 2.8s to 0s)
     - mobile: save 10.5s (reduce estimated savings from 10.5s to 0s)


# Test for [article page](http://test.twreporter.org:3000/a/russian-invasion-of-ukraine-2022-kid-frequently-asked-questions)

## Before

### Mobile
image
[article-mobile.pdf](https://github.com/twreporter/twreporter-npm-packages/files/9651193/article-mobile.pdf)

### Desktop
image
[article-desktop.pdf](https://github.com/twreporter/twreporter-npm-packages/files/9651204/article-desktop.pdf)

## After

### Mobile
image
[article-after-mobile.pdf](https://github.com/twreporter/twreporter-npm-packages/files/9651793/article-after-mobile.pdf)

### Desktop
image
[article-after-desktop.pdf](https://github.com/twreporter/twreporter-npm-packages/files/9652042/article-after-desktop.pdf)


# Test for [index page](http://test.twreporter.org:3000/)

## Before

### Mobile
image
[homepage-mobile.pdf](https://github.com/twreporter/twreporter-npm-packages/files/9652482/homepage-mobile.pdf)

### Desktop
image
[homepage-desktop.pdf](https://github.com/twreporter/twreporter-npm-packages/files/9652522/homepage-desktop.pdf)


## After

### Mobile
image
[homepage-after-mobile.pdf](https://github.com/twreporter/twreporter-npm-packages/files/9652372/homepage-after-mobile.pdf)

### Desktop
image
[homepage-after-desktop.pdf](https://github.com/twreporter/twreporter-npm-packages/files/9652435/homepage-after-desktop.pdf)
